src/value.rs

```diff
@@ -159,6 +160,24 @@ impl Clone for Value {
     }
 }
 
+fn display_vec<T: fmt::Display>(
+    head: &str,
+    f: &mut fmt::Formatter<'_>,
+    v: &[T],
```

Comment:
I might miss this in some places, but mutable output variables should be the last variables in a function.

```suggestion
    v: &[T],
    f: &mut fmt::Formatter<'_>,
```